README.md

```diff
@@ -83,7 +83,22 @@ When deploying the multi cluster traffic controller using the make targets the f
     ```sh
     (export $(cat ./controller-config.env | xargs) && export $(cat ./aws-credentials.env | xargs) && make build install run
     ```
-## 3. Run the ingress agent locally
+
+## 3. Deploy the agent
+1. Update the secret in `config/agent/secret.yaml` with the correct credentials for the control plane. (**N.B.** The server should remain `https://mctc-control-plane-control-plane:6443`)
```

Comment:
I wonder if we could automate this step in a similar way to how it's done here:
https://github.com/Kuadrant/multi-cluster-traffic-controller/blob/main/hack/.argocdUtils

Not necessarily as part of this PR though

Reply:
yeah, I'd really love to get this automated, and out of the path of accidental commits.